File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/UserLocationSearchResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class UserLocationSearchResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public bool HasMoreResults { get; set; }
    public int Count { get; set; }
    public List<UserLocationListResponse> UserLocations { get; set; }
    public string Returning { get; set; }
}

File: server-scala-play-server/app/model/UserLocationSearchResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for UserLocationSearchResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class UserLocationSearchResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  hasMoreResults: Option[Boolean],
  count: Option[Int],
  userLocations: Option[List[UserLocationListResponse]],
  returning: Option[String]
)

object UserLocationSearchResponse {
  implicit lazy val userLocationSearchResponseJsonFormat: Format[UserLocationSearchResponse] = Json.format[UserLocationSearchResponse]
}

